WalletWasabi.Fluent/ViewModels/OpenDirectory/OpenConfigFileViewModel.cs

```diff
@@ -17,6 +18,16 @@ namespace WalletWasabi.Fluent.ViewModels.OpenDirectory
 	public partial class OpenConfigFileViewModel : TriggerCommandViewModel
```

Comment:
I meant to have an `OpenTextFileViewModel:TriggerCommandViewModel` inherited class and just set the filepath in the superclass.

Reply:
If you mean to have one generic class for file opening, that would not work.

All the classes have `NavigationMetaData` attribute. where properties are set like `Keyword`, `Title` and so. These are mostly needed for `Action Center`.

```
[NavigationMetaData(
		Title = "Config File",
		Caption = "",
		Order = 4,
		Category = "Open",
		Keywords = new[]
		{
			"Browse", "Open", "Config", "File"
		},
		IconName = "document_regular")]
```

That is why there are three different classes for a "seemingly" same logic.

Reply:
No, I've meant to have the following:

```
public class OpenFileViewModel : TriggerCommandViewModel
{
public virtual string FilePath
public override ICommand TargetCommand =>
}
```

and having 

```
OpenLogsViewModel: OpenFileViewModel 
{
 override FileName => Services.Config.FilePath
}
```

Reply:
fixed